src/views/User.js

```diff
@@ -67,22 +67,18 @@ export const User = () => {
     const [order, setOrder] = useState("asc");
     const [orderBy, setOrderBy] = useState("USER_NAME");
     const [userData, setUserData] = useState([]);
-    
     const [time, setTime] = useState(0);
-
-    // state to check stopwatch running or not
     const [isRunning, setIsRunning] = useState(false);
+    const firstUpdate = useRef(true); 
+    const delayTime = 5; // 5/10 of a second; the delay between users's last keystroke and the filtering API call
```

Comment:
this is half a second?   this isn't measured in milliseconds?

Reply:
Yep, so this is half a second. We're using `setInterval()` to do the "delay": `setInterval(() => setTime(time + 1), 100);`. That code runs every 100 milliseconds. Every 100 milliseconds, we increment time by 1, meaning that when time == 10, that is a full second (there are 1000 milliseconds in 1 second). Thus, 5 is half a second.

Reply:
okay, then we need a good name for that...   `delayTime_in_10ths_of_a_second` ?

Reply:
No need for the `10ths` here. We can do a little math to convert whatever config value into the necessary precision.

Reply:
so... we're back to `delayTime_in_milliseconds` and we'll do the math in the app...

Reply:
or... `REACT_APP_FILTER_TIME_IN_MILLISECONDS` at the top-level?

Reply:
thinking we could use the same setting for any page with filtering and keystrokes...

Reply:
ohhh i really like that idea. added!